tests/bin/check_tron_jobs_test.py

```diff
@@ -91,59 +91,141 @@ def test_get_relevant_action_picks_the_one_that_failed(self):
                 'job_name': 'MASTER.kwatest',
             },
         ]
-        actual = check_tron_jobs.get_relevant_action(action_runs, State.FAILED)
+        actual = check_tron_jobs.get_relevant_action(
+            action_runs, State.FAILED, {}
+        )
         assert_equal(actual["state"], "failed")
 
     def test_get_relevant_action_picks_the_first_one_succeeded(self):
         action_runs = [
             {
-                'id': 'MASTER.test.action1',
-                'action_name': 'action1',
-                'state': 'succeeded',
-                'start_time': time.localtime(time.time() - 1200),
+                'id':
```

Comment:
I don't usually nitpick on style, but can you put this back? Or is the auto-formatting doing this? This is not "normal" k/v style

Reply:
Yeah, it's a little weird. Yapf does this.